contracts/Contribution.sol

```diff
@@ -139,14 +140,31 @@ contract Contribution is Controlled, TokenController {
 
   // ETH-AIX exchange rate
   function exchangeRate() constant public initialized returns (uint256 rate) {
+    
     if (getBlockTimestamp() <= startTime + 1 hours) {
       // 15% Bonus
       rate = 2300;
     } else if (getBlockTimestamp() <= startTime + 2 hours) {
       // 10% Bonus
       rate = 2200;
-    } else {
+    } else if((getBlockTimestamp() > startTime + 2 hours) && (getBlockTimestamp() < startTime + 1 days)) {
       rate = 2000;
+    } else {
+      if (notCollectedAmountAfter24Hours == 0) {
+        notCollectedAmountAfter24Hours = weiToCollect();
+      }
+      uint256 twentyPercentFromSupply = notCollectedAmountAfter24Hours.mul(20).div(100);
+      uint256 thirtyPercentFromSupply = notCollectedAmountAfter24Hours.mul(30).div(100);
+      uint256 firstDayRaised = totalWeiCap.sub(notCollectedAmountAfter24Hours);
+      uint256 weiTilBonusTwentyExist = firstDayRaised.add(twentyPercentFromSupply);
+      uint256 weiTilBonusThirtyExist = firstDayRaised.add(thirtyPercentFromSupply);
+      if(totalWeiCollected <= weiTilBonusTwentyExist ) {
+        rate = 2300;
+      } else if(totalWeiCollected > weiTilBonusTwentyExist && totalWeiCollected <= weiTilBonusThirtyExist) {
+        rate = 2200;
+      } else {
+        rate = 2000;
+      }
```

Comment:
What happens if I invest the first 40%?
It should be `2300`for the first 20% and `2200` the next 10% and `2000` for the final 10%.

Reply:
I'll merge and make a new PR to have a look at this